src/actions.js

```diff
@@ -103,7 +139,7 @@ export function fetchOrganisations() {
 
           dispatch(
             addNotification(
-              `Organisation "${firstOrganisation.name}" selected`,
+              `Organisation "${(firstOrganisation && firstOrganisation.name) || "none"}" selected`,
```

Comment:
Putting this comment here because only this line is changed, but: if we enter the above block, then an organisation is chosen there, and then we choose one here again. I think this block should be removed.

Reply:
true, hopefully solved in above comment